hwdb/parse_hwdb.py

```diff
@@ -117,6 +117,8 @@ def property_grammar():
              ('KEYBOARD_LED_NUMLOCK', Literal('0')),
              ('KEYBOARD_LED_CAPSLOCK', Literal('0')),
              ('ACCEL_MOUNT_MATRIX', mount_matrix),
+             ('ID_INPUT_TABLET', Literal('0')),
+             ('ID_INPUT_TOUCHSCREEN', Literal('1')),
```

Comment:
this is not generic enough, if we add this, it should work generically to set 0 or one on any ID_INPUT_foo tag, not just allow this exact combination.  Excluded is the ID_INPUT_TOUCHPAD_INTEGRATION tag tough.